File: Frontend/trackr-administrador/src/app/shared/dtos/seguridad/usuario-expediente-sidebar-dto.ts
import { ExpedientePadecimientoSidebarDTO } from "./expediente-padecimiento-sidebar-dto";

export class UsuarioExpedienteSidebarDTO{
    idUsuario:number;
    nombreCompleto:string;
    imagenBase64?:string;
    tipoMime?: string;
    idGenero:number;
    edad:string;
    colonia?:string;
    ciudad?:string;
    estado?:string;
    padecimientos?: ExpedientePadecimientoSidebarDTO[];
}

File: Backend/TrackrAPI/TrackrAPI/Dtos/GestionExpediente/UsuarioExpedienteSidebarDTO.cs

namespace TrackrAPI.Dtos.GestionExpediente;

public class UsuarioExpedienteSidebarDTO
{
    public int IdUsuario { get; set; }
    public string NombreCompleto { get; set; }
    public string? TipoMime { get; set; }
    public string? ImagenBase64 { get; set; }
    public string Genero { get; set; }
    public string Edad { get; set; }
    public string? Colonia { get; set; }
    public string? Ciudad { get; set; }
    public string? Estado { get; set; }
    public IEnumerable<ExpedienteSidebarDTO>? Padecimientos { get; set; }
}

